rules/github-scrum-workflow/rule.md

```diff
@@ -0,0 +1,79 @@
+---
+type: rule
+archivedreason:
+title: How to set up GitHub scrum workflow
+guid: e63da194-c5d2-4830-9106-d164aafcddb5
+uri: do-you-know-the-how-to-set-up-github-workflow
+created: {date_created}
+authors: []
+related:
+
+---
+### How to set up GitHub scrum workflow
+
+There are several ways of turning your GitHub repository into a simple but powerful agile work environment, the following is the recommended:
+
+<!--endintro-->
+
+#### How it works
+
+- Product Backlog Items (PBIs) are reported as **issues**
+- Points and meta data are assigned to items as **labels**
+- **milestones** are used to group issues in Sprints
+
+#### 1. Create issues as backlog items
+
+To create a new backlog item, just create a new issue.
```

Comment:
I'd recommend setting up GH Issue templates for this https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

Reply:
> I'd recommend setting up GH Issue templates for these

Added instruction to configure issue templates